about/press.html

```diff
@@ -102,12 +121,17 @@
       <p>For press inquiries, send us an email to <a href="mailto:summer-of-code@railsgirls.com">summer-of-code@railsgirls.com</a>.</p>
 
       <br>
+      {% for link in page.links %}
+        <a href="#{{ link.href }}"><i class="fa fa-{{ link.icon }}"></i>{{ link.title }}</a>
```

Comment:
super small change: could you add a space after the `</i>`? that way the icon and text aren't "squished" together. :D